streams/catalog/src/main/java/com/hortonworks/streamline/streams/catalog/exception/ServiceComponentNotFoundException.java

```diff
@@ -32,7 +28,7 @@ public ServiceComponentNotFoundException(Throwable cause) {
         super(cause);
     }
 
-    public ServiceComponentNotFoundException(Long clusterId, ServiceConfigurations service, ComponentPropertyPattern component) {
-        this(String.format("Component [%s] not found for service [%s] in cluster with id [%d]", component.name(), service.name(), clusterId));
+    public ServiceComponentNotFoundException(Long clusterId, String serviceName, String componentName) {
```

Comment:
I think I answered the comment earlier too. It's due to avoid cross reference between modules.